src/common/components/ExternalLink.js

```diff
@@ -14,6 +14,7 @@ const ExternalLink = ({ href, component, children, linkProps }) => {
     window.open(href, '_blank', 'noopener,noreferrer');
     trackEvent('Outbound Link: Click', { props: { url: href } });
     event.preventDefault();
+    event.stopPropagation();
```

Comment:
question: do we need both `preventDefault` and `stopPropagation`? Usually one is enough.

Reply:
@paulschreiber This was an improvement I added that was needed on an old change, so this can be removed, but I think this is still useful, so the difference between `preventDefault` and `stopPropagation` is that `preventDefault` stops any other action link to the specific element, `stopPropagation` prevent other actions linked to parent elements, so I think this is still relevant for this component

Reply:
How is `stopPropagation` needed? `preventDefault` should prevent the link from opening on its own.

Reply:
@paulschreiber Right now, it is not needed, it was needed when inside a button there was a need for a <ExternalLink> in that case this was needed to avoid the button from triggering when the users clicks the link